06-building-a-program-start-to-finish/writing-tests/output/test.js

```diff
@@ -0,0 +1,136 @@
+const invertedIndex = require('../src/inverted-index.js');
+const books = require('./books');
+
+describe('Inverted index Suite', () => {
+   // Create an instance of the Index class
+  const newIndex = new invertedIndex();
+  const sampleSentence = 'As &you can see here, you, have defined the function';
+  beforeAll(() => {
+    newIndex.createIndex('books', books);
+  });
+
+  describe('Class Inverted Index', () => {
+    it('should be a class', () => {
+      expect(newIndex instanceof Object).toBe(true);
+      expect(newIndex instanceof invertedIndex).toBe(true);
+      expect(typeof newIndex).toBe('object');
+    });
+  });
+
+  describe('Tokenize String', () => {
+    it('should be available in class InvertedIndex', () => {
+      expect(invertedIndex.tokenize).toBeDefined();
+    });
+    it('should return an array containing alphabets only', () => {
+      expect(invertedIndex.tokenize(sampleSentence)).toMatch(/[A-z]+$/);
+    });
+    it('should return an array containing alphabets only', () => {
+      expect(invertedIndex.tokenize(sampleSentence)).not.toMatch(/&/);
+    });
+    it('should return an array containing alphabets only', () => {
+      expect(invertedIndex.tokenize(sampleSentence)).toEqual(['as',
+        'can', 'defined', 'function', 'have', 'here', 'see', 'the',
+        'you', 'you']);
+    });
+    it('should return an array containing the correct number of words', () => {
+      expect(invertedIndex.tokenize(sampleSentence).length).toBe(10);
+    });
+  });
+
+  describe('Unique Words', () => {
+    it('should be available in class InvertedIndex', () => {
+      expect(invertedIndex.uniqueWords).toBeDefined();
+    });
+    it('should return an array of words without duplicates', () => {
+      expect(invertedIndex.uniqueWords(sampleSentence).length).toBe(9);
+    });
+    it('should return an array of words without duplicates', () => {
+      expect(invertedIndex.uniqueWords(sampleSentence)).toEqual(['as',
+        'can', 'defined', 'function', 'have', 'here', 'see', 'the',
+        'you']);
+    });
+  });
+
+  describe('Populate Index', () => {
```

Comment:
I believe this is where you should've called `createIndex`, since you're testing that the index is created correctly.